akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -619,12 +621,14 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
         log.debug("Inbound channel is now active")
       } else if (status == ChannelEndpointStatus.ERRORED) {
         areonErrorLog.logErrors(log, 0L)
+        Try(stopMediaDriver())
```

Comment:
this was not the issue I could repeat, but should be good to try stopping it and deleting files in this case also

Reply:
Should we at least log the exception, if any, even though we otherwise ignore it?

Reply:
That would probably have to be println then, this is in the startup phase of the actor system. I don't think it will be able to throw, just added Try to be on the safe side. Can add catch-println inside stopMediaDriver instead.